libraries/AP_AccelCal/AP_AccelCal.cpp

```diff
@@ -71,34 +71,42 @@ void AP_AccelCal::update()
                 if (step != _step) {
                     _step = step;
 
-                    const char *msg;
-                    switch (step) {
-                        case 1:
-                            msg = "level";
-                            break;
-                        case 2:
-                            msg = "on its LEFT side";
-                            break;
-                        case 3:
-                            msg = "on its RIGHT side";
-                            break;
-                        case 4:
-                            msg = "nose DOWN";
-                            break;
-                        case 5:
-                            msg = "nose UP";
-                            break;
-                        case 6:
-                            msg = "on its BACK";
-                            break;
-                        default:
-                            fail();
-                            return;
+                    if (_use_gcs_snoop) {
+                        const char *msg;
+                        switch (step) {
+                            case 1:
+                                msg = "level";
```

Comment:
They can be an enum, but this was there already, I didn't touch it other than add an *if*